### PR TITLE
Resolve issue #443

### PR DESCRIPTION
**Pull request by cawerkenthin**
Resolve issue #443 - Satisfied session id in case of NotApplicable moveOn

### CODE REVIEW

File: cmi5_spec.md [hunk 1]
```diff
@@ -745,6 +745,9 @@ Regardless of the verbs the AUs use in statements, the LMS MUST record and provi
 <th align="left" nowrap>LMS Obligations</th><td>
 The LMS MUST use the "Satisfied" statement when the learner has met the moveOn criteria of all AU's in a block.  In this statement the LMS MUST use the block id (Section 13.1.2) as the Object id (Section 9.4 - Object) and use "https://w3id.org/xapi/cmi5/activitytype/block" as the value of the "type" property in the Object's Definition.<br>
 <br>
+For all "Satisfied" statements triggered as a result of an AU launch session, the LMS MUST use the session id from the AU launch in the statements.
+<br>
+<br>
```
Comment (brianjmiller) [inline cmi5_spec.md]: This table cell feels a little disjointed to me. Does it make more sense to move this line to the bottom (or perhaps the top) and move the last sentence of the current last paragraph, specifically:

> The LMS SHOULD NOT issue multiple statements with "Satisfied" for the same Block or Course within a course registration for a given learner.

Onto a line by itself similar to how this one is. To be clear, I'd do this cell as:

> The LMS MUST use the "Satisfied" statement when the learner has met the moveOn criteria of all AU's in a block. In this statement the LMS MUST use the block id (Section 13.1.2) as the Object id (Section 9.4 - Object) and use "https://w3id.org/xapi/cmi5/activitytype/block" as the value of the "type" property in the Object's Definition.
> 
> The LMS MUST also use the "Satisfied" statement when the learner has met the moveOn criteria for all AU's in a course. In this statement the LMS MUST use the course id (Section 13.1.1) as the Object id (Section 9.4 - Object) and use "https://w3id.org/xapi/cmi5/activitytype/course" as the value of the "type" property in the Object's Definition. 
>
> For all "Satisfied" statements triggered as a result of an AU launch session, the LMS MUST use the session id from the AU launch in the statements. 
> 
> The LMS SHOULD NOT issue multiple statements with "Satisfied" for the same Block or Course within a course registration for a given learner.

File: cmi5_spec.md [hunk 2]
```diff
@@ -905,7 +908,9 @@ Sample JSON:
 
 <a name="registration"></a> 
 ###9.6.1 registration
-The value for the registration property used in the context object MUST be the value provided by the LMS. The LMS MUST generate this value and pass it to the AU via the launch URL.  At the time of registration, the LMS MUST evaluate MoveOn criteria in the course structure.  For example, Blocks with MoveOn criteria of "Not Applicable" in the course structure would be evaluated and could generate Satisfied statements at this time.
+The value for the registration property used in the context object MUST be the value provided by the LMS. The LMS MUST generate this value and pass it to the AU via the launch URL.  At the time of registration, the LMS MUST evaluate MoveOn criteria in the course structure.  For example, Blocks with MoveOn criteria of "Not Applicable" in the course structure would be evaluated and could generate Satisfied statements at this time.<br>
+<br>
```
Comment (brianjmiller) [inline cmi5_spec.md]: HTML line breaks shouldn't be necessary (or preferred) in a markdown document. Can we remove these for the same effect visually?

File: cmi5_spec.md (hunk 1 above)
Comment (brianjmiller) [inline cmi5_spec.md]: Shouldn't need HTML line breaks in this markdown.

**Comment by cawerkenthin** [comment]
@brianjmiller, 
There are a huge number of br tags throughout the spec.  Perhaps we should have a separate issue to remove them all?

**Comment by brianjmiller** [comment]
In some cases they can be necessary, though I've not checked whether the remaining ones are only the necessary ones. I know the xAPI spec had a pass done on its markdown in general, we could consider doing the same.

**Pull request merged**